### DISCUSSION TITLE
DAD in the Address resolution section

### DISCUSSION THREAD

**Discussion by EduardVasilenko** [becarpenter/book6#32] [General]
Hi Brian,
It is very good that the discussion about multicast issues is so deep.
But the primary source for multicast is DAD which has not been mentioned in the section. Especially since DAD is needed for every address that the IPv6 interface has.
People may not understand why NS is so harmful.
Eduard

**Comment by becarpenter** [comment]
DAD is mentioned in the next section (Auto-config). I also point to draft-thubert-6man-ipv6-over-wireless which really explains the problem space. I wish the IETF would adopt that instead of pretending everything is OK (when the network at every IETF meeting proves that it isn't).

**Comment by EduardVasilenko** [comment]
It looks like you have a strategy to avoid any duplication. It is the second time you answer "look in the different section".
But then the person would not be capable to read just one section to understand everything related. It contradicts another goal that you have.

**Comment by becarpenter** [comment]
Fair comment. In this particular topic, it seems that everything is connected to everything. For now I will add a note at the beginning of the chapter that one should read all the sections (and some extra cross-reference perhaps).

**Comment by EduardVasilenko** [comment]
Truly speaking, I am not sure what is better:
- to keep everything lean/deduplicated
- to make every section self-reliant.
It needs marketing research - what people like more.
My personal preference does not matter - I am strange enough.

**Comment by timchown** [comment]
Single sources of truth, then no need to update 10 sections if something changes.

On 24 Nov 2022, at 11:33, Eduard Vasilenko ***@***.***> wrote:



Truly speaking, I am not sure what is better:

  *   to keep everything lean/deduplicated
  *   to make every section self-reliant.
It needs marketing research - what people like more.
My personal preference does not matter - I am strange enough.

**Comment by becarpenter** [comment]
> Single sources of truth, then no need to update 10 sections if something changes.
Ideally, yes, but it does need a lot of cross-references.

**Comment by becarpenter** [comment]
Added a mention of DAD in the [Address resolution](https://github.com/becarpenter/book6/blob/main/2.%20IPv6%20Basic%20Technology/Address%20resolution.md) section.
